charts/celeborn/templates/worker-statefulset.yaml

```diff
@@ -67,6 +67,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if or .Values.priorityClass.worker.name .Values.priorityClass.worker.create }}
```

Comment:
Should be consistent with the logic used to create the priority class resource, i.e. use `.Values.priorityClass.worker.create` to control.

Reply:
What if user want use a predefined priorityClass?

For example, I want use `system-cluster-critical` for celeborn master Pods and `system-node-critical` for celeborn worker Pods. These priorityClasses shipped with k8s by default.

Reply:
Sounds reasonable, thanks for answering doubt!